src/IntegerDivisionTable.cpp

```diff
@@ -9,7 +9,7 @@
  * expensive method, while the compile-time set uses
  * the cheapest method for the given divisor.
  */
-#include <stdint.h>
+#include <cstdint>
```

Comment:
Seems like this should include IntegerDivisionTable.h (maybe instead of cstdint). It would give errors if the declarations of these arrays happened to not match the definitions.